pilot/pkg/bootstrap/certcontroller.go

```diff
@@ -43,9 +44,10 @@ const (
 var (
 	// dnsCertDir is the location to save generated DNS certificates.
 	// TODO: we can probably avoid saving, but will require deeper changes.
-	dnsCertDir  = "./var/run/secrets/istio-dns"
-	dnsKeyFile  = path.Join(dnsCertDir, "key.pem")
-	dnsCertFile = path.Join(dnsCertDir, "cert-chain.pem")
+	dnsCertDir = "./var/run/secrets/istio-dns"
+
+	dnsKeyFile  = "./" + filepath.Join(dnsCertDir, "key.pem")
```

Comment:
nit: why do we need ` "./" + ` won't it end up like `././var...`?

Reply:
`filepath.Join` calls `filepath.Clean` which eliminates the leading `./`.  

Did we want absolute or relative paths for these base directories? Relative makes testing a bit easier (var can be in the CWD).

Reply:
Relative so we can run locally